core/fuzz/fuzz_targets/fuzz_reader_fs.rs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Arbitrary, Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use opendal::raw::oio::ReadExt;
+use opendal::{services, Operator};
+use std::env;
+
+#[derive(Debug)]
+enum ReaderAction {
+    Read { size: usize },
+    Seek { offset: usize, mode:SeekMode },
```

Comment:
Code seems not formatted by `rustfmt`.

Reply:
And I prefer to have a `Seek(SeekFrom)` instead.

Reply:
solved by https://github.com/apache/incubator-opendal/commit/27d5bd0807ce9435396b86033a5961c56b9835b4